src/client/swagger/unity.yml

```diff
@@ -625,6 +805,12 @@ components:
           type: string
           enum: [aws, gcm, azure]
           example: gcm
+        subscriberId:
+          type: string
+          description: TODO - not sure what this is referring to
+        status:
+          type: string
+          description: TODO - not sure what this is referring to
```

Comment:
@randycoulman which of the marketplace subscription properties should we consider as required?

Reply:
Not sure; I think we only get those if there is actually a marketplace subscription and not for Zuora. Maybe @ischolten knows better than I do?

Reply:
For the operator ui, the marketplaceSubscription was optional on an account. If there is a marketplaceSubscription, it will have subscriberId, status, and marketplace. These fields are just used as information to display on the operator ui so we know if its a marketplace account and additional information for investigation if need be.

Also, it looks like this `Marketplace` is actually a representation of a `MarketplaceSubscription`.